src/Sulu/Bundle/MigrationBundle/Migrations/Version20221122205424.php

```diff
@@ -0,0 +1,824 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sulu\Bundle\MigrationBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20221122205424 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->skipIf(
+            !$this->connection->getDatabasePlatform() instanceof \Doctrine\DBAL\Platforms\PostgreSQL100Platform,
+            "Migration can only be executed safely on '\Doctrine\DBAL\Platforms\PostgreSQL100Platform'."
+        );
```

Comment:
Just example migration file which should be replaced in future into more splitted files.